docs/using-radicle/creating-and-sharing-projects.md

```diff
@@ -29,17 +29,18 @@ At the moment, [projects can only be created through the Upstream client][fa].
 ## Sharing projects
 
 Projects can be shared with their Radicle ID. The Radicle ID can be copied from
-the project page.
+the project page (see above).
 
-Copy and pasting a project's Radicle ID into your search bar will find it on
-the network for you. Following a project will replicate it to your local
-machine, making it available to all of your connected peers. To learn more
-about following & replication, [click here][un].
+Copy and pasting a project's Radicle ID into your search bar will find it on the
+network for you. Following a project will replicate it to your local machine,
+making it available to all of your connected peers. To learn more about
+following & replication, [See][un].
```

Comment:
```suggestion
following & replication, [see][un].
```

Reply:
Had to add a word here, so fixing it in my next commit